content/docs/04-clusters/02-data-center/01-maas/02-architecture.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Architecture"
+metaTitle: "MAAS Architecture with Palette"
+metaDescription: "Learn about the architecture used to support MAAS using Palette"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# MAAS Bare-Metal Kubernetes Architecture
+
+The following are some architectural highlights of bare-metal Kubernetes clusters that Palette deploys using Canonical's MAAS.  Canonical MAAS is an open-source tool that lets you discover, commission, deploy, and dynamically reconfigure a large network of individual units.
+
+- Palette supports the use of MAAS with Spectro Cloud’s open source Cloud Native Computing Foundation (CNCF) Cluster API that supports Canonical MAAS: https://github.com/spectrocloud/cluster-api-provider-maas.
```

Comment:
The URL should be in markdown style instead of the current raw style.

Reply:
Already fixed :)